src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/components/SettingsIndex.tsx

```diff
@@ -144,7 +153,11 @@ const SettingsIndex = () => {
                     onClick={() => redirectToRoute('pim_enrich_locale_index')}
                     content={
                       countEntities.hasOwnProperty('count_locales')
-                        ? translate('pim_settings.count.locales', {count: countEntities['count_locales']})
+                        ? translate(
+                            'pim_settings.count.locales',
+                            {count: countEntities['count_locales']},
+                            countEntities['count_locales']
+                          )
```

Comment:
Maybe it will be easier to read with a simple helper that returns the translation for a given `countEntities` property, so that you can reuse it on all entities?

```suggestion
                    content={getPluralizedTranslation('pim_settings.count.locales', 'count_locales')}
```